src/use-dropdown-menu.ts

```diff
@@ -37,7 +37,6 @@ export default function useDropdownMenu(itemCount: number) {
 
 		// If the menu is currently open focus on the first item in the menu
 		if (isOpen) {
-			itemRefs.current[0].current.focus();
```

Comment:
Why is this line being removed?

Reply:
This seems to be an outdated line from before the function `moveFocus` was implemented. If you look at the proceeding line, it is called with an argument of `0` which accomplishes exactly what line 40 was doing. It could be accomplished in a different pull request but since it was a one line change I thought I would toss it in this PR.